workflow-bot-app/src/test/resources/application-test.yaml

```diff
@@ -1,3 +1,7 @@
+wdk:
```

Comment:
do we need a configuration class to hold these wdk configs ? i didn't an existing one.
the workflows section should be moved inside as well.